tests/expected/decode/test.tar.stdout

```diff
@@ -27,19 +27,19 @@
 │           │   │   │   └── string <- repeat base.ascii-char.octal := "000000"
 │           │   │   ├── prefix <- slice 155 tar.ascii-string.opt0 := ""
 │           │   │   └── pad <- repeat-count 12 [= 0]
-│           │   └── file <- { ... } :=
-│           │       ├── 0 := 0
-│           │       ├── 1 := 5
-│           │       ├── 2 := 22
-│           │       ├── 3 := 7
-│           │       ├── 4 := 0
-│           │       ├── 5 := 2
-│           │       ├── 6 := 0
-│           │       ├── 7 := 0
-│           │       ├── 8 := 77
-│           │       ├── 9 := 97
+│           │   └── file <- repeat-count (header.size) base.u8 :=
+│           │       ├── 0 <- base.u8 := 0
```

Comment:
Because this is a `repeat-count`, we already know what the type of each index will be. How do we rank the quality of this changed output, compared to before (even if it is more of a side-effect than a conscious decision)?

Reply:
Ah good point, it's a side-effect but might be worth checking to see if we can do better.